Vagrantfile

```diff
@@ -20,17 +20,39 @@ Vagrant.configure(2) do |config|
   # `vagrant box outdated`. This is not recommended.
   # config.vm.box_check_update = false
 
+
   # Create a forwarded port mapping which allows access to a specific port
   # within the machine from a port on the host machine. In the example below,
   # accessing "localhost:8080" will access port 80 on the guest machine.
+
+  tomcatPort = ENV['TOMCAT_PORT']
+  if tomcatPort.nil?
+    tomcatPort = '8888'
+  end
+
+  transPort = ENV['NODEJS_PORT']
+  if transPort.nil?
+    transPort = '8094'
+  end
+
+  mergePort = ENV['P2P_PORT']
+  if mergePort.nil?
+    mergePort = '8096'
+  end
+
+  mapnikPort = ENV['NODE_MAPNIK_SERVER_PORT']
+  if mapnikPort.nil?
+    mapnikPort = '8000'
+  end
```

Comment:
It seems like we're adding yet another place where ports can be configured, or is this meant to be *the* place to do it.  This would make four different methods we use to configure web service ports, three of which I've documented in #468.